### PR TITLE
feat(deploy): full deployment of spinnaker on GCE

### PR DESCRIPTION
**Pull request by lwander**
This deploys (but doesn't support updating yet) a full, clustered spinnaker on GCE. It also includes some startup tweaks that need to be baked into our images.

### CODE REVIEW

File: halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/google/GoogleGateService.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    result.add(getVaultClientService());
+    return result;
+  }
+
+  @Override
+  public Settings buildServiceSettings(DeploymentConfiguration deploymentConfiguration) {
+    Settings settings = new Settings(deploymentConfiguration.getSecurity().getApiSecurity());
+    settings.setArtifactId(getArtifactId(deploymentConfiguration.getName()))
+        .setAddress(buildAddress())
+        .setLocation("us-central1-f")
```
Comment (jtk54) [inline halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/google/GoogleGateService.java]: Would probably want this configurable in the future.

File: halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/google/GoogleOrcaBootstrapService.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@EqualsAndHashCode(callSuper = true)
+@Data
+@Component
+public class GoogleOrcaBootstrapService extends OrcaBootstrapService implements GoogleDistributedService<OrcaBootstrapService.Orca> {
+  final DeployPriority deployPriority = new DeployPriority(6);
```
Comment (jtk54) [inline halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/google/GoogleOrcaBootstrapService.java]: Is there a priority for each service? Or is this more of a "they should deploy/start in roughly this order"?

File: halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/google/GoogleClouddriverBootstrapService.java [hunk 3]
```diff
@@ -25,6 +25,7 @@
 import com.netflix.spinnaker.halyard.deploy.spinnaker.v1.service.distributed.SidecarService;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import lombok.experimental.Delegate;
```
Comment (jtk54) [inline halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/google/GoogleClouddriverBootstrapService.java]: Heh `experimental`, should be good right?

File: halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/google/GoogleGateService.java (hunk 1 above)
Comment (lwander) [inline halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/google/GoogleGateService.java]: Yup

File: halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/google/GoogleOrcaBootstrapService.java (hunk 2 above)
Comment (lwander) [inline halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/google/GoogleOrcaBootstrapService.java]: Priority probably isn't the best word - it's the latter. Let me know if you have ideas for better naming.

File: halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/google/GoogleClouddriverBootstrapService.java (hunk 3 above)
Comment (lwander) [inline halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/google/GoogleClouddriverBootstrapService.java]: Works pretty well :)

**Pull request merged**